backend/package.json

```diff
@@ -6,26 +6,28 @@
   "main": "src/app.ts",
   "type": "module",
   "scripts": {
-    "dev": "concurrently \"tsc --watch\" \"nodemon dist/app.js\"",
+    "dev": "concurrently \"tsc --watch\" \"nodemon dist/backend/src/app.js\"",
```

Comment:
might need to adjust tsconfig if it's copying the whole tree like this

not critical though, I'll check it out later!